pset-iterators.js

```diff
@@ -115,6 +144,25 @@ console.log('Problem 4:')
 
 // Add your code below this line
 
+
+const presidentsBetween1850and1900 = presidents.filter(function (president){
+
+ if ((president.took_office.substring(0, 4) >= "1850") && (president.took_office.substring(0, 4) < "1900")) {
+
+//   console.log(president.took_office.substring(0, 4))
+  return president
+
+ } else {
+//   console.log("add nothing")
+  return
+ }
+
+
+})
+
+console.log(presidentsBetween1850and1900)
```

Comment:
@thetripler Good work here utilizing the filter method in your solution here. Nice use of the conditional there to check for presidents who served between 1850 and 1900 and return those presidents. One suggestion for your ` if/else` statement since you are not returning anything in your `else`, you can remove the `else` statement.

Take a look at this approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)